app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio"%>
+  </div>
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true, autocomplete: "photo" %>
+  </div>
+  <div class="field">
+    <%= f.label :email %><br />
```

Comment:
- [ ] The views in devise for the registration and login must be modified to match the wireframes and your styling.
You can take a look at the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project.

- Your Design:

![Screenshot from 2022-12-23 00-16-52](https://user-images.githubusercontent.com/79058364/209235088-b5b39a26-58a9-42b7-a864-48b637485e45.png)


- Original template:

![image](https://user-images.githubusercontent.com/79058364/208264719-cde14266-91f0-41af-9259-5993f6715d34.png)